webui/react/src/services/api.ts

```diff
@@ -80,33 +81,58 @@ export const getExperimentDetails =
 export const getTrialDetails =
   generateApi<TrialDetailsParams, TrialDetails>(Config.getTrialDetails);
 
-export const killExperiment = generateDetApi<KillExpParams, Api.V1KillExperimentResponse, void>(
-  Config.killExperiment,
+export const forkExperiment = generateApi<ForkExperimentParams, number>(Config.forkExperiment);
+
+export const activateExperiment = generateDetApi<
+ExperimentIdParams,
+Api.V1ActivateExperimentResponse,
+void
+> (
+  Config.activateExperiment,
 );
 
-export const forkExperiment = generateApi<ForkExperimentParams, number>(Config.forkExperiment);
+export const archiveExperiment = generateDetApi<
+ExperimentIdParams,
+Api.V1ArchiveExperimentResponse,
+void
+> (
+  Config.archiveExperiment,
+);
 
-export const patchExperiment = generateApi<PatchExperimentParams, void>(Config.patchExperiment);
+export const cancelExperiment = generateDetApi<
+ExperimentIdParams,
+Api.V1CancelExperimentResponse,
+void
+> (
+  Config.cancelExperiment,
+);
 
-export const archiveExperiment = async (id: number, archive = true): Promise<void> => {
-  try {
-    await archive ?
-      Config.detApi.Experiments.determinedArchiveExperiment(id) :
-      Config.detApi.Experiments.determinedUnarchiveExperiment(id);
-  } catch (e) {
-    processApiError('archiveExperiment', e);
-    throw e;
-  }
-};
+export const killExperiment = generateDetApi<
+ExperimentIdParams,
+Api.V1KillExperimentResponse,
+void
+> (
+  Config.killExperiment,
+);
 
-export const setExperimentState = async (
-  { state, ...rest }: PatchExperimentState,
-): Promise<void> => {
-  return await patchExperiment({
-    body: { state },
-    ...rest,
-  });
-};
+export const pauseExperiment = generateDetApi<
+ExperimentIdParams,
+Api.V1PauseExperimentResponse,
+void
+> (
```

Comment:
the line breaks here look a bit weird but I'm not sure we can format it in a different way. this works : )

@hkang1  any ideas?

Reply:
That would work?
![Uploading Schermata 2020-11-10 alle 07.52.54.png…]()

Reply:
I'll see if there's a tslint we can auto lint something like this... What you suggested @apizzini works for me.

Reply:
looks like breaking up generics into multiple lines is not an easy thing without hacking things together:
https://github.com/typescript-eslint/typescript-eslint/issues/455

Reply:
Changed, should be good.